xinference/model/image/stable_diffusion/core.py

```diff
@@ -40,7 +41,7 @@ def __init__(
         self._kwargs = kwargs
 
     def load(self):
-        import torch
+        # import torch
```

Comment:
Does it need to be uncommented?

Reply:
The linter is complaining about it. I checked the following code and found only some commented code referenced it, so I kept it as commented instead of removed.